### PR TITLE
docs: fix UI should follow the design # 9

### LINKED ISSUE #9: fix UI not as expected
should follow designer UI

### PR DESCRIPTION
**Pull request by nstksean**
## #9 Make UI follow the design

# PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

# PR Type

What kind of change does this PR introduce?

- [X] Bugfix
 - [ ] Feature
 - [ ] Code style update (formatting, local variables)
 - [ ] Refactoring (no functional changes, no api changes)
- [ ]  Build related changes
 - [ ] CI related changes
 - [ ] Documentation content changes
 - [ ] angular.io application / infrastructure changes
 - [ ] Other... Please describe:
 
# What is the current behavior?
[95042ab](https://github.com/nstksean/doeYu/pull/10/commits/95042ab41b5467174ac73eb9bfcc21d8424d0435)

# What is the new behavior?
Check all component UI in homePage, make it closer to design layout.

# Does this PR introduce a breaking change?
- [ ] Yes
- [X] No

## Other information
[Design layout](https://www.figma.com/file/1iOEdPgCgV6v7bVRblnWF6/Week-1-%E5%8F%B0%E7%81%A3%E6%97%85%E9%81%8A%E6%99%AF%E9%BB%9E%E5%B0%8E%E8%A6%BD?node-id=60%3A10463) Page mock up 首頁/日間模式 component 人氣美食 is not center alignment.

### CODE REVIEW

**Comment by lucifiel0121** [comment]
What is the current behavior?
What is the new behavior?

這兩條不可能沒有，不然等於你沒寫 code
如果是ＵＩ改動 你可以描述 UI change ，比如附上截圖
描述清楚你要改動的部分是哪邊，也讓相關人員比較好確認是否修復

**Comment by lucifiel0121** [comment]
[fix:fix nav bar to designer UI](https://github.com/nstksean/doeYu/pull/10/commits/ef6784c558d3d076db47eeca912578624be2f150)

  
[fix:fix UI in home page](https://github.com/nstksean/doeYu/pull/10/commits/7a9555bde0a753ed15aa4af99e8ac3275dbbaf97)


附上這兩條 commit 修復後的截圖，
這邊特別要求一張正常截圖，一張要蓋原設計圖
因為昨天發現你目前的ＵＩ和設計圖差有點多

**Comment by lucifiel0121** [comment]
如果你修復問題，需要比較多條 commit
link 也可以寫在 ＰＲ 不一定是寫在 commit

**Comment by lucifiel0121** [comment]
另外這屬於 Bugfix

Refactoring 會偏向之前說改寫 useContext -> redux 這類
no functional changes, no api changes 指的是你程式優化架構或是語法寫法，但不能改壞原本功能。

**Comment by lucifiel0121** [comment]
閱讀後請在修改完成的對應 comment 丟一個 emoji

**Review by nstksean — COMMENTED** [review]
navBarOrigin (p.1)
navBarFixed(p.2)
Provide fixed UI and UI cover by design. Remove search bar/I18n button/Theme changer.

**Review by nstksean — COMMENTED** [review]
subscribeOrigin(p.1)
subscribeFixed(p.2)
Provide subscribe UI(p.1) and subscribe UI cover by design(p.2).

**Review by nstksean — COMMENTED** [review]
homePageOrigin(p.1)
homePageFixed(p.2)
Provide homePage UI fixed and homePage UI cover by design.

**Comment by lucifiel0121** [comment]
正確的 UI 會跟設計圖重合 
你要再確定數值 調整到重合

目前看起來出了 nav 其他都與設計圖不符合

**Comment by lucifiel0121** [comment]
正確的樣式會比較類似 https://github.com/nstksean/doeYu/pull/11

另外要注意 這兩個分支應當是不同的，小心改a壞b的狀況

即這個pr只修復無關rwd的樣式

**Comment by lucifiel0121** [comment]
修正後還是要附修正後的圖

**Review by nstksean — COMMENTED** [review]
fixUiForMyBest
人氣美食 in design layout is not center alignment, so I make it align.

File: src/components/NavBar/navbar.module.css [hunk 1]
```diff
@@ -4,16 +4,17 @@
     width: 100%;
     height: 100px;
```
Comment (lucifiel0121) [inline src/components/NavBar/navbar.module.css:5]: fix(UI): fix nav bar

**Pull request merged**